### PR TITLE
Fix eo filter bug

### PR DESCRIPTION
**Pull request by ASmithOWL**
Adds logic to ensure that bridge filters are sorted properly given the update in the `EoServer` logic for acquiring logs from the Eo Contract

### CODE REVIEW

File: crates/node/src/main.rs [hunk 1]
```diff
@@ -362,15 +362,15 @@ fn setup_eo_server(
...
         .address(vec![contract_address])
         .topics(blob_settled_topic.clone(), None, None, None)
         .build();
 
     let bridge_filter = web3::types::FilterBuilder::default()
-        .from_block(BlockNumber::Number(75127.into()))
-        .to_block(BlockNumber::Number(75127.into()))
+        .from_block(BlockNumber::Number(312.into()))
+        .to_block(BlockNumber::Number(312.into()))
         .address(vec![contract_address])
```
Comment (mitch-vrrb) [inline crates/node/src/main.rs:374]: is this just a patch fix and we will get this into an env/config variable later?

**Review by mitch-vrrb — APPROVED** [review]

File: crates/node/src/main.rs (hunk 1 above)
Comment (eureka-cpu) [reply crates/node/src/main.rs:374]: Yeah it's been talked about

**Review by eureka-cpu — APPROVED** [review]
:shipit:

File: crates/node/src/main.rs (hunk 1 above)
Comment (eureka-cpu) [reply crates/node/src/main.rs:374]: #158

**Comment by eureka-cpu** [comment]
@ASmithOWL can you merge main into this so that it doesn't cause merge conflicts for the pending upstream PRs?

**Pull request merged**